samples/hello-world/src/main/java/com/samples/helloworld/app/RootActivity.kt

```diff
@@ -0,0 +1,33 @@
+package com.samples.helloworld.app
+
+import android.os.Bundle
+import android.view.ViewGroup
+import com.badoo.ribs.android.RibActivity
+import com.badoo.ribs.core.Rib
+import com.badoo.ribs.core.modality.BuildContext.Companion.root
+import com.samples.helloworld.R
+import com.samples.helloworld.hello_world.HelloWorld
+import com.samples.helloworld.hello_world.builder.HelloWorldBuilder
+
+class RootActivity : RibActivity() {
+
+    private val deps = object : HelloWorld.Dependency {
+        override val name: String = "world"
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        setContentView(R.layout.activity_root)
+        super.onCreate(savedInstanceState)
+    }
+
+    override val rootViewGroup: ViewGroup
+        get() = findViewById(R.id.root)
+
+    override fun createRib(savedInstanceState: Bundle?): Rib {
+        val helloWorldBuilder: HelloWorldBuilder = HelloWorldBuilder(deps)
+        val helloWorld: HelloWorld = helloWorldBuilder.build(
+            root(savedInstanceState)
+        )
+        return helloWorld
+    }
```

Comment:
Let's collapse this to:

```suggestion
    override fun createRib(savedInstanceState: Bundle?): Rib =
        HelloWorldBuilder(deps).build(
            root(savedInstanceState)
        )
```

Reply:
Applied in 62b2ab220124dc01d6089f1834149085ee7d8a46